qiita_db/support_files/patches/python_patches/30.py

```diff
@@ -0,0 +1,78 @@
+from qiita_db.sql_connection import TRN
+from qiita_db.metadata_template.constants import (NA_VALUES, TRUE_VALUES,
+                                                  FALSE_VALUES)
+from qiita_db.metadata_template import SampleTemplate, PrepTemplate
+from qiita_db.study import Study
+
+bool_vals = set(TRUE_VALUES + FALSE_VALUES + [None])
```

Comment:
It is considered for the bool col check though, since a bool col can have True, False, NULL.